src/test/java/com/dougnoel/sentinel/elements/ElementTests.java

```diff
@@ -179,4 +181,33 @@ public void sendSpecialKey() {
 		assertTrue("Expecting key press input to be empty.", input.getText().isEmpty());
 	}
 	
+	@Test
+	public void checkParentColorsOfTransparentElement() {
+		JavascriptExecutor js = (JavascriptExecutor) WebDriverFactory.getWebDriver(); 
+		
+		Color endingColor = Color.blue;
+		String hexOfEndingColor = "#"+Integer.toHexString(endingColor.getRGB()).substring(2);
+		
+		BaseSteps.navigateToPage("TextboxPage");
+		js.executeScript("arguments[0].style.backgroundColor=arguments[1];", getElement("Body").element(), hexOfEndingColor);
+		Assert.assertEquals(endingColor, getElement("Car Checkbox").getBackgroundColor());
```

Comment:
Change the sndingColorVariable to something like colorBlue and hexOfEndingColor to whatever it is, the blueValue? Variable names should be descriptive, not enigmatic.

Reply:
Understood. Partially a relic of the original assertions when two were present. I've modified it accordingly and made the variable names more understandable